README.md

```diff
@@ -350,6 +349,20 @@ for details and use-cases.
     Alice     = "Bob"
   }
   ```
+#### Cognito User Pool Domain
+
+- **`domain_prefix`**: *(Optional `string`)*
+
+  Type a domain prefix to use for the sign-up and sign-in pages that are hosted by Amazon Cognito,
+  e.g. `https://{YOUR_PREFIX}.auth.eu-west-1.amazoncognito.com`. The prefix must
+  be unique across the selected AWS Region. Domain names can only contain
+  lower-case letters, numbers, and hyphens. Default is `""`.
+
+- **`domain_certificate_arn `**: *(Optional `string`)*
+
+  The ARN of an ISSUED ACM certificate in us-east-1 for a custom domain.
+  Default is `null`.
```

Comment:
null should not be communicated to the user... "default is not to use a custom domain"

Reply:
resolved in 3b4e65b